lib/bojangles.rb

```diff
@@ -104,16 +104,22 @@ def go!
     errors = DepartureComparator.compare
     current_time = Time.now
     new_errors, resolved_errors = compare_errors(errors, cached_error_messages)
+    issue_numbers = cached_error_messages
```

Comment:
So, this variable contains more than just the "numbers", right?

Reply:
Yes, it's a mapping from message text to GH issue numbers.